sklearn/neighbors/lof.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from .base import NeighborsBase
+from .base import KNeighborsMixin
+from .base import UnsupervisedMixin
+
+from ..metrics import pairwise_distances
+
+from ..utils import check_array
+from ..utils import _get_n_jobs
+
+__all__ = ["LOF"]
+
+
+class LOFMixin(object):
+    """Mixin for Local-Outlier-Factor computation"""
+
+    def k_distance(self, X=None):
+        """
+        Compute the k_distance and the neighborhood of querry samples X wrt
```

Comment:
This isn't a terribly informative docstring. It doesn't define what k_distance is, or what self._fit_X is. Maybe change it to describe things in terms of the algorithm, irrespective of the underlying implementation. Also, `query` not `querry`.